djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
+
+    if version == 'latest':
+        version = None
+    else:
+        if not date_re.match(version):
+            raise ImproperlyConfigured(
```

Comment:
I had actually considered that but I'm on the fence about it.  We always grab the API version at start-up when `stripe_objects` is evaluated, so although it is runtime, it's also at start-up.  I suppose the user *might* update the value later, although I'm not really sure why they'd want to. It also means separating the validation logic for the value so there's two places that examine the value of `STRIPE_API_VERSION`.